src/main/java/lotto/domain/WinningPrize.java

```diff
@@ -4,7 +4,7 @@ public enum WinningPrize {
     THREE(3, 5000),
     FOUR(4, 50000),
     FIVE(5, 1500000),
```

Comment:
```suggestion
    FOUR(4, 50_000),
    FIVE(5, 1_500_000),
```

다른 값들도 구분이 가능하지 않을까요 ㅎㅎ 일관성을 맞추면 좋을 것 같아요